src/api/window/window.ts

```diff
@@ -517,8 +480,74 @@ export class _Window extends EmitterBase {
      * @property {string} state - The preload script state:
      "load-failed", "failed", "succeeded"
      */
+
+    private nativeWindow: any;
+
     constructor(wire: Transport, public identity: Identity) {
         super(wire);
+
+        // if it's openfin environment, need to add a native window to current window object
+        if (this.isOpenFinEnvironment()) {
+            if (this.wire.environment.isWindowExists(identity.uuid, identity.name)) {
+                if (identity.name === window.name) {
+                    this.nativeWindow = window;
+                }
+            }
+        }
+    }
+
+    // create a new window
+    public createWindow(options: any): Promise<_Window> {
```

Comment:
should this be private ?

Reply:
@rdepena This method belongs to _window class, but it's called in _windowModule class.  If it's private, it's not accessible.

Reply:
gotcha